acceptance/tests/vault/vault_test.go

```diff
@@ -111,6 +114,16 @@ func TestVault(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, caConfig.Provider, "vault")
 
+	// Validate that consul sever is running correctly and the consul members command works
```

Comment:
@ishustava If this would be useful somewhere else as a generic helper then I could put it somewhere?

Reply:
I think it's ok for now. If we end up using it in multiple places, then we can put it in helpers